drivers/overlay/overlay.go

```diff
@@ -1530,9 +1530,6 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 	}
 
 	composefsLayers := filepath.Join(workDirBase, "composefs-layers")
```

Comment:
This definition is now pretty far from the only point where it gets referenced.  Can it be moved?

Reply:
sure, fixed now